exercises/change/canonical-data.json

```diff
@@ -37,6 +37,12 @@
                 "expected": [2, 2, 5, 20, 20, 50, 100, 100, 100, 100, 100, 100, 100, 100, 100]
             },
             {
+                "description": "posible change without one coins available",
```

Comment:
@weapp just a small grammatical nit pick, `coins` should be singular instead of plural.  I like the added twist of not having a penny available.  Thanks